### PR TITLE
Replaces nearly every mention of the SCCV Horizon being a Station with Ship/Vessel/Facility Dynamically V2

### PR DESCRIPTION
**Pull request by Chada-1**
AFAIK the SCCV Horizon is a ship or vessel, it wouldn't be considered a Station, and as it stands lots of people are having problems moving away from the old terminology due to how everything on the map is named, so this is an attempt to rename everything to suit the new map faaar more, it hopefully doesn't break anything, but I need help making sure it doesn't.

I used Wildkins' dynamic var for many uses of station, but some just wouldn't work with the var due to loading *before* the map, so they'd runtime. This means I had to generalize a lot of phrasings to exclude Station, using vessel or facility, which are both vague enough to be a station or a ship.

This is also sort of a continuation of this PR: #13940

### CODE REVIEW

**Comment by Chada-1** [comment]
!review

File: code/controllers/subsystems/news.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	NEW_SS_GLOBAL(SSnews)
 
 /datum/controller/subsystem/news/Initialize(timeofday)
-	CreateFeedChannel("Station Announcements", "Automatic Announcement System", 1, 1, "New Station Announcement Available")
+	CreateFeedChannel("Standard Announcements", "Automatic Announcement System", 1, 1, "New Standard Announcement Available")
```
Comment (alsoandanswer) [inline code/controllers/subsystems/news.dm:18]: Could do away with "Standard" altogether.

Comment (Chada-1) [reply code/controllers/subsystems/news.dm]: Alright, done.

File: config/example/config.txt [hunk 2]
```diff
@@ -1,620 +0,0 @@
-## Server name: This appears at the top of the screen in-game. In this case it will read "tgstation: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice
```
Comment (Greenjoe12345) [inline config/example/config.txt:1]: should this be being deleted...?

File: maps/_common/areas/station/storage.dm [hunk 3]
```diff
@@ -55,5 +55,5 @@
 	icon_state = "storage"
 
 /area/storage/shields
-	name = "Station Shield Control"
+	name = "Vessel Shield Control"
```
Comment (Greenjoe12345) [inline maps/_common/areas/station/storage.dm:58]:
```suggestion
	name = "[current_map.station_type] Shield Control"
```

File: maps/_common/mapsystem/map.dm [hunk 4]
```diff
@@ -84,21 +84,21 @@
...
 	//event messages
 
-	var/meteors_detected_message = "A meteor storm has been detected on collision course with the station. Estimated three minutes until impact, please activate station shields, and seek shelter in the central ring."
+	var/meteors_detected_message = "A meteor storm has been detected on collision course with the vessel. Estimated three minutes until impact, please activate station shields, and seek shelter in the central ring."
 	var/meteor_contact_message = "Contact with meteor wave imminent, all hands brace for impact."
-	var/meteor_end_message = "The station has cleared the meteor shower, please return to your stations."
+	var/meteor_end_message = "The vessel has cleared the meteor shower, please return to your stations."
 
-	var/ship_meteor_contact_message = "The NDV Icarus reports that it has downed an unknown vessel that was approaching your station. Prepare for debris impact - please evacuate the surface level if needed."
+	var/ship_meteor_contact_message = "The NDV Icarus reports that it has downed an unknown vessel that was approaching your vessel. Prepare for debris impact - please evacuate the surface level if needed."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:91]:
```suggestion
	var/ship_meteor_contact_message = "Sensors indicate that wreckage from a derelict ship is approaching the [current_map.station_type]. Prepare for debris impact - please evacuate the surface level if needed."
```
we're not near the Icarus now so should say something else, perhaps?

File: maps/_common/mapsystem/map.dm [hunk 5]
```diff
@@ -84,21 +84,21 @@
...
 	var/ship_detected_end_message = "Ship debris colliding now, all hands brace for impact."
-	var/ship_meteor_end_message = "The last of the ship debris has hit or passed by the station, it is now safe to commence repairs."
+	var/ship_meteor_end_message = "The last of the ship debris has hit or passed by the vessel, it is now safe to commence repairs."
 
-	var/dust_detected_message = "A belt of space dust is approaching the station."
-	var/dust_contact_message = "The station is now passing through a belt of space dust."
-	var/dust_end_message = "The station has now passed through the belt of space dust."
+	var/dust_detected_message = "A belt of space dust is approaching the vessel."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:95]:
```suggestion
	var/dust_detected_message = "A belt of space dust is approaching the [current_map.station_type]."
```

File: maps/_common/mapsystem/map.dm [hunk 6]
```diff
@@ -84,21 +84,21 @@
...
+	var/dust_contact_message = "The vessel is now passing through a belt of space dust."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:96]:
```suggestion
	var/dust_contact_message = "The [current_map.station_type] is now passing through a belt of space dust."
```

File: maps/_common/mapsystem/map.dm [hunk 7]
```diff
@@ -84,21 +84,21 @@
...
+	var/dust_end_message = "The vessel has now passed through the belt of space dust."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:97]:
```suggestion
	var/dust_end_message = "The [current_map.station_type] has now passed through the belt of space dust."
```

File: maps/_common/mapsystem/map.dm [hunk 8]
```diff
@@ -84,21 +84,21 @@
...
 
-	var/radiation_detected_message = "High levels of radiation detected near the station. Please evacuate into one of the shielded maintenance tunnels."
-	var/radiation_contact_message = "The station has entered the radiation belt. Please remain in a sheltered area until we have passed the radiation belt."
-	var/radiation_end_message = "The station has passed the radiation belt. Please report to medbay if you experience any unusual symptoms. Maintenance will lose all-access again shortly."
+	var/radiation_detected_message = "High levels of radiation detected near the vessel. Please evacuate into one of the shielded maintenance tunnels."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:99]:
```suggestion
	var/radiation_detected_message = "High levels of radiation detected near the [current_map.station_type]. Please evacuate into one of the shielded maintenance tunnels."
```

File: maps/_common/mapsystem/map.dm [hunk 9]
```diff
@@ -84,21 +84,21 @@
...
+	var/radiation_contact_message = "The vessel has entered the radiation belt. Please remain in a sheltered area until we have passed the radiation belt."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:100]:
```suggestion
	var/radiation_contact_message = "The [current_map.station_type] has entered the radiation belt. Please remain in a sheltered area until we have passed the radiation belt."
```

File: maps/_common/mapsystem/map.dm [hunk 10]
```diff
@@ -84,21 +84,21 @@
...
+	var/radiation_end_message = "The vessel has passed the radiation belt. Please report to medbay if you experience any unusual symptoms. Maintenance will lose all-access again shortly."
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:101]:
```suggestion
	var/radiation_end_message = "The [current_map.station_type] has passed the radiation belt. Please report to medbay if you experience any unusual symptoms. Maintenance will lose all-access again shortly."
```

File: maps/_common/mapsystem/map.dm [hunk 11]
```diff
@@ -84,21 +84,21 @@
...
 
 	var/list/rogue_drone_detected_messages = list("A combat drone wing operating out of the NDV Icarus has failed to return from a sweep of this sector, if any are sighted approach with caution.",
```
Comment (Greenjoe12345) [inline maps/_common/mapsystem/map.dm:103]: Icarus mentions again

File: maps/aurora/aurora-1_centcomm.dmm [hunk 12]
```diff
@@ -22577,7 +22577,7 @@
 /area/shuttle/merchant)
 "kFK" = (
 /obj/machinery/computer/security/telescreen{
-	desc = "Connects to a station's emergency camera networks after the station sends out a distress signal.";
+	desc = "Connects to a facilities emergency camera networks after the facility sends out a distress signal.";
```
Comment (Greenjoe12345) [inline maps/aurora/aurora-1_centcomm.dmm:22580]:
```suggestion
	desc = "Connects to a facility's emergency camera networks after the facility sends out a distress signal.";
```

File: config/example/config.txt (hunk 2 above)
Comment (Chada-1) [reply config/example/config.txt:1]: No it shouldn't, thank you.

**Comment by Chada-1** [comment]
Closing in hindsight because I'm worried about creating a dissociation between the announcer audio and the text. If we decide to do this in the future, the PR will still be here.

**Comment by Greenjoe12345** [comment]
Weren't a number of the audio files redone for the  horizon any way?

**Pull request closed without merging**